modules/lakeformation/variable.tf

```diff
@@ -0,0 +1,9 @@
+variable "name" {
+  description = "The name of the database."
+  type        = "string"
+}
+
+variable "iam_roles" {
+  description = "The arn of the segment datalake iam role."
```

Comment:
You probably need to update the description here since now the type has changed from `string` to `map(string)`